packages/stark-core/src/modules/http/helpers/http-helpers.ts

```diff
@@ -0,0 +1,45 @@
+import { HttpParameterCodec, HttpParams } from "@angular/common/http";
+import { StarkQueryParam } from "@nationalbankbelgium/stark-core";
+import { CustomEncoder } from "../entities/custom-encoder";
+
+/**
+ *  @ignore
+ */
+const _reduce: Function = require("lodash/reduce");
+
+/**
+ * A custom implementation of HttpParameterCodec to correctly encode query parameters.
+ * @link https://github.com/NationalBankBelgium/stark/issues/1130
+ */
+export const CUSTOM_ENCODER: HttpParameterCodec = new CustomEncoder();
+
+/**
+ * Converts the Map<string, StarkQueryParam> required by the service into a HttpParams object required by Angular
+ * @param starkQueryParam - params to convert
+ */
+export function convertStarkQueryParamsIntoHttpParams(starkQueryParam: Map<string, StarkQueryParam>): HttpParams {
```

Comment:
I agree this function is something that can be re-used, however I think we should put it as a static method in a Util class (like we the ones we have under `packages/stark-core/src/util/`). This way this can be also used by developers when dealing with `StarkQuerrParams`

Reply:
So should i move it to there or create a util class specific to http?

Reply:
Indeed, we should create a Util class specific for Http ;)